pkg/clusters/addons/kong/kong.go

```diff
@@ -146,17 +166,39 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 		)
 	}
 
+	if err := clusters.CreateNamespace(ctx, cluster, a.namespace); err != nil {
+		return err
+	}
+
 	// do the deployment and install the chart
 	args := []string{"--kubeconfig", kubeconfig.Name(), "install", DefaultDeploymentName, "kong/kong"}
-	args = append(args, "--create-namespace", "--namespace", a.namespace)
-	args = append(args, a.deployArgs...)
+	args = append(args, "--namespace", a.namespace)
+	if a.enterprise {
+		if err := deployKongEnterpriseLicenseSecret(ctx, cluster, a.namespace, KongLicenseSecretName); err != nil {
+			return err
+		}
+
+		if err := prepareSecrets(ctx, a.namespace); err != nil {
+			return err
+		}
+
+		args = append(args, "--version", "2.3.0", "-f", "https://raw.githubusercontent.com/Kong/charts/main/charts/kong/example-values/minimal-k4k8s-with-kong-enterprise.yaml")
+		args = append(args, "--set", "admin.type=LoadBalancer", "--set", "admin.annotations.konghq.com/protocol=http", "--set", "enterprise.rbac.enabled=true",
+			"--set", "env.enforce_rbac=on", "--set", "ingressController.enabled=false",
+			"--set", "ingressController.installCRDs=false", "--set", "env.kong_password=password",
+			"--skip-crds")
+	} else {
+		args = append(args, a.deployArgs...)
+	}
+
 	stderr = new(bytes.Buffer)
+	fmt.Printf("deployment args %s", args)
 	cmd = exec.CommandContext(ctx, "helm", args...)
 	cmd.Stdout = io.Discard
 	cmd.Stderr = stderr
 	if err := cmd.Run(); err != nil {
 		if !strings.Contains(stderr.String(), "cannot re-use") { // ignore if addon is already deployed
-			return fmt.Errorf("%s: %w", stderr.String(), err)
+			return fmt.Errorf(" %s: %w", stderr.String(), err)
```

Comment:
Was this space accidentally added?

```suggestion
			return fmt.Errorf("%s: %w", stderr.String(), err)
```